pages/cfp.md

```diff
@@ -6,13 +6,13 @@ permalink: /participate/
 
 ### Dates
 
-7th Jun to 23rd Jun
+13th May to 28th May
```

Comment:
Can we drop these commits from the dependency PR?

Reply:
On it.

Reply:
Done.